crs_support/launch/environment_launch.launch.py

```diff
@@ -98,11 +98,39 @@ def generate_launch_description():
         ])
     '''
 
+    process_planner_test_server = launch_ros.actions.Node(
+        node_executable='crs_motion_planning_process_planner_test',
+        package='crs_motion_planning',
+        node_name='process_test_server',
+        output='screen')
+
     test_process_planner = launch_ros.actions.Node(
```

Comment:
Does this run during the main application?

Reply:
Test_process_planner does not run on the main application, I left the defined launch variable there so I could easily add it back for testing. I can completely remove it though.

Reply:
It's fine if it stays if it doesn't interfere